toolchain/check/member_access.cpp

```diff
@@ -300,19 +313,24 @@ static auto LookupMemberNameInScope(Context& context, SemIR::LocId loc_id,
             context.facet_types().Get(facet_type->facet_type_id);
         // Witness that `T` implements the `*assoc_interface`.
         SemIR::InstId witness_inst_id = SemIR::InstId::None;
-        for (auto base_interface : facet_type_info.impls_constraints) {
+        // TODO: This assumes `impls_constraints` are in the same order as
+        // `CompleteFacetType::required_interfaces`, but there may be more
```

Comment:
```suggestion
        // `CompleteFacetType::required_interfaces`, but once we add
        // support for named constraints there may be more
```

Reply:
Reworded to say that we require the named constraint ones to come later.